playbooks/byo/openshift-cluster/openshift-logging.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Add a comment that this playbook is a preview of upcoming changes and to see inventory/byo/hosts.*.example for details of the currently supported method?